lib/service/servicecfg/config_test.go

```diff
@@ -533,3 +534,106 @@ func TestValidateConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyEnabledService(t *testing.T) {
+	tests := []struct {
+		desc             string
+		config           *Config
+		errAssertionFunc require.ErrorAssertionFunc
+	}{
+		{
+			desc: "auth enabled",
+			config: &Config{
+				Auth: AuthConfig{
+					Enabled: true,
+				},
+			},
+			errAssertionFunc: require.NoError,
+		},
```

Comment:
This seems super verbose for this simple logic it's testing, I'm tempted to suggest something like the following if it works but it's probably way less readable so I'll leave it up to you

```go
cfg := &Config{}
tests := []struct{
	name string
	enabled *bool
}{
	{"auth", &cfg.Auth.Enabled},
	{"ssh", &cfg.SSH.Enabled},
	...
}
```

Reply:
How about I one-line all of these instead? That way it doesn't seem so overly dense.